### ISSUE TITLE
Keyboard accessibility - cannot TAB to other tabs

### ISSUE THREAD

**Issue by MarcusJT** [github/tab-container-element#20]
I've played with the demo at https://github.github.com/tab-container-element/examples/ and found that in its initial state I can press TAB to focus each of the tabs at the top, but once I activate a tab (ENTER) it ceases to be possible to TAB to the other tabs, they can only be reached via the arrow keys (and only when the tab bar has focus) which is quite confusing.

**Comment by koddsson** [comment]
Thanks for the issue @MarcusJT!

I just made changes to the example page and the `README.md` that added `tabindex="-1"` to the tabs that was missing. This change makes sure that the inactive tabs cannot be tabbed to but are still activated by the arrow keys.
